From 8859f257c2e4468875665d8782812b83f8ca741f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Tue, 5 Mar 2024 15:10:45 +0200
Subject: [PATCH] ptp: Don't install test executable

And handle it like all our other test executables.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/6262>
---
 subprojects/gstreamer/libs/gst/helpers/ptp/meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/libs/gst/helpers/ptp/meson.build b/subprojects/gstreamer/libs/gst/helpers/ptp/meson.build
index 55f1c6a106e..e12ca3fe73d 100644
--- a/libs/gst/helpers/ptp/meson.build
+++ b/libs/gst/helpers/ptp/meson.build
@@ -112,9 +112,7 @@ exe_test = executable('gst-ptp-helper-test', 'main.rs',
   override_options : ['rust_std=2018'],
   rust_args : ['--test', rust_args],
   dependencies : [cap_dep],
-  link_with : conf,
-  install_dir : helpers_install_dir,
-  install : true)
+  link_with : conf)
 
 test('gst-ptp-helper-test', exe_test, protocol : 'rust')
 
-- 
GitLab

